stable/postgresql/templates/statefulset.yaml

```diff
@@ -79,6 +79,86 @@ spec:
           mountPath: {{ .Values.persistence.mountPath }}
           subPath: {{ .Values.persistence.subPath }}
       {{- end }}
+      {{- if .Values.initContainer.masterEnabled }}
+            - name: custom-init-container
+        image: {{ template "postgresql.initContainer.image" . }}
+        imagePullPolicy: "{{ .Values.initContainer.image.pullPolicy }}"
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+      {{- if .Values.initContainer.command }}
+        command:
+          {{- range .Values.initContainer.command }}
+          - {{ . }}
+          {{- end }}
+        {{- end}}
+        securityContext:
+          runAsUser: {{ .Values.volumePermissions.securityContext.runAsUser }}
+        volumeMounts:
+        - name: data
+          mountPath: {{ .Values.persistence.mountPath }}
+        env:
```

Comment:
Why do you need all the env. variables to be passed to the master initcontainer but not to the slave one?

Reply:
Yeah, I guess forgot to add those to slaves.